### PR TITLE
fix orgs-state-selector UI

### PR DESCRIPTION
**Pull request by achen2401**
address issues found related to this story:  https://jira.movember.com/browse/TN-247

regrouped organization entries on ui based on parent org per state
fixed consent modal text to reference consent organization's short name 
fixed indentations in code

### CODE REVIEW

File: portal/templates/profile_macros.html [hunk 1]
```diff
@@ -714,89 +714,119 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
...
+                                            });
+                                        };
+                                    };
+                                    return s;
+                                })(parentId);
+
+                                contentHTML = "<div class='radio'>" +
+                                            "<label><input class='clinic' type='radio' id='%orgId%_org' value='%orgId%' state='%state%' name='organization' data-parent-name='%parentOrgName%' data-parent-id='%parentOrgId%'>%translatedOrgName%</label>" +
+                                            "</div>";
+                                contentHTML = contentHTML.replace(/\%orgId\%/g, item.id)
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Could you use the more standard `{{var}}` or `{var}`? Characters that match in sets are a little easier to read and grep for.

File: portal/templates/profile_macros.html [hunk 2]
```diff
@@ -714,89 +714,119 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
...
-                      //case of pre-selected clinic, need to check if any clinic has prechecked
-                      setTimeout(function() { handleCheckedItem();}, 500);
-                      $("#userOrgs input[name='organization']").each(function() {
-                        if ($(this).val() != 0) OT.getDefaultModal(this);
                       });
+                    
+                    var selectOptions = $("#stateSelector").sortOptions();
+                    if (selectOptions.length > 0) {
+                        $("#stateSelector").empty().append(selectOptions)
+                          .append("<option value='none'>Other</option>")
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Please tag with translation call :)

File: portal/templates/profile_macros.html [hunk 3]
```diff
@@ -714,89 +714,119 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
...
+                          .prepend("<option value='' selected>Select</option>")
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Please tag with translation call :)

File: portal/templates/profile_macros.html [hunk 4]
```diff
@@ -881,7 +913,7 @@ <h4 class="modal-title">{{ _("Consent to share information") }}</h4>
                         </div>
                         <input type="hidden" class="agreement-url" value="{{consent_agreement.url|safe}}" />
                         <br/>
-                        <p>{{_("I consent to sharing information with the ")}}<span class="consent-clinic-name">{{consent_agreement.organization_name}}.</span></p>
+                        <p>{{_("I consent to sharing information with ")}}<span class="consent-clinic-name">{{consent_agreement.organization_shortname}}</span>.</p>
```
Comment (ivan-c) [inline portal/templates/profile_macros.html]: Please use a variable here, for clarity. eg:
```
{% trans org_shortname=consent_agreement.organization_shortname %}
I consent to sharing information with {{org_shortname}}
{% endtrans %}
```

File: portal/static/js/main.js [hunk 5]
```diff
@@ -1868,7 +1868,15 @@ OrgTool.prototype.populateOrgsList = function(items) {
         };
         if (item.extension) orgsList[item.id].extension = item.extension;
         if (hasValue(item.language)) orgsList[item.id].language = item.language;
-        if (item.identifier) orgsList[item.id].identifier = item.identifier;
+        if (item.identifier) {
+            orgsList[item.id].identifier = item.identifier;
+            (item.identifier).forEach(function(identifier) {
+                if (identifier.system === "http://us.truenth.org/identity-codes/shortname") {
```
Comment (ivan-c) [inline portal/static/js/main.js]: That string might be better as a global variable/constant (if you're using it more than once)

**Comment by achen2401** [comment]
@ivan-c ivan, i think i addressed all your requests.  let me know if you see any issue with my changes.

**Pull request merged**